test/governance/utils/Votes.test.js

```diff
@@ -0,0 +1,41 @@
+const { expectRevert } = require('@openzeppelin/test-helpers');
+
+const { expect } = require('chai');
+
+const Votes = artifacts.require('VotesMock');
+
+contract('Voting', function (accounts) {
```

Comment:
We are missing tests for a lot of the functionality and a lot of cases. As guidelines: we should have tests for all non-private functions, and for each of them there should be a test for every `require` on top of testing the basic functionality.

In this particular case most things are really covered in the tests for ERC721Votes. But they should be tested here as well. I think we should have a Votes "behavior" that we can use both for Votes on its own as well as ERC721Votes, and possibly also ERC20Votes. Maybe you can take the current tests from ERC721Votes and wrap them in a function that we can use in all those places? At least in Votes and ERC721Votes.